src/main/res/menu/nav_drawer.xml

```diff
@@ -16,5 +16,20 @@
         android:title="@string/filter_import_export"
         android:icon="@drawable/ic_import_export_white_24dp"
         app:showAsAction="always"
-        />
+        >
```

Comment:
Put a menu inside an item, and android takes care of inflating the menu on click for you. Voila! Who knew?